src/test/java/core/basesyntax/service/FileReaderImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String FILE_EXIST = "src/test/resources/input.csv";
+    private static final String WRONG_FILE_PATH = "src/test/wrongPath/input.csv";
+    private static FileReader fileReader;
+
+    @BeforeClass
+    public static void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void readFile_WrongPath_notOk() {
```

Comment:
```suggestion
    public void readFile_wrongPath_notOk() {
```

read [this message](https://mateacademy.slack.com/archives/C040S2B99FH/p1667833485897189) and check it everywhere